index.html

```diff
@@ -10,17 +10,24 @@
 
         <div id="container">
             <div id="email-nav">
-                <span class="email-nav_inbox">Inbox</span>
+                <span class="email-nav_inbox nav--selected">Inbox</span>
```

Comment:
I haven't see a `--` in a css class before. Is that a thing? I'd get rid of it and make it single dash if it isn't